frontend/appflowy_tauri/src/appflowy_app/views/DocumentPage.hooks.ts

```diff
@@ -7,33 +7,40 @@ import {
 import { useParams } from 'react-router-dom';
 import { DocumentData } from '../interfaces/document';
 import { DocumentController } from '$app/stores/effects/document/document_controller';
-
+import { useAppDispatch } from '../stores/store';
 
 export const useDocument = () => {
   const params = useParams();
-  const [ documentId, setDocumentId ] = useState<string>();
-  const [ documentData, setDocumentData ] = useState<DocumentData>();
-  const [ controller, setController ] = useState<DocumentController | null>(null);
+  const [documentId, setDocumentId] = useState<string>();
+  const [documentData, setDocumentData] = useState<DocumentData>();
+  const [controller, setController] = useState<DocumentController | null>(null);
+  const dispatch = useAppDispatch();
 
   useEffect(() => {
+    let document_controller: DocumentController | null = null;
```

Comment:
```suggestion
    let documentController: DocumentController | undefined;
```
camel case

Reply:
You are right, I forgot to switch styles when writing rust and ts